src/include/daos_security.h

```diff
@@ -474,6 +474,24 @@ daos_ace_to_str(struct daos_ace *ace, char *buf, size_t buf_len);
 int
 daos_acl_from_strs(const char **ace_strs, size_t ace_nr, struct daos_acl **acl);
 
+/**
+ * Convert an Access Control List (daos_acl) to a list of Access Control Entries
+ * formatted as strings.
+ *
+ * Each entry in ace_strs is dynamically allocated. So is the array itself. It
+ * is the caller's responsibility to free all of them.
+ *
+ * \param[in]	acl		DAOS ACL
```

Comment:
nit: alignment?

Reply:
Hmm, I think Github is doing something weird with the tabs in comments, specifically. I wonder if there's some way to fix this so it matches what our IDEs are all set to.